float32[20] joint_pos